Targets/AppStoreConnectUI/Sources/Authorization/AuthView.swift

```diff
@@ -6,27 +6,49 @@
 //
 
 import SwiftUI
+import UniformTypeIdentifiers
 
-struct AuthView: View {
-    @StateObject var viewModel: AuthViewModel
+public struct AuthView: View {
+    @StateObject private var viewModel: AuthViewModel
+    @State private var isImporting: Bool = false
     
-    var body: some View {
+    public init(viewModel: AuthViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
+    }
+    
+    public var body: some View {
         VStack(alignment: .center) {
             VStack(alignment: .leading) {
                 AuthInputField(title: "Issuer ID", binding: $viewModel.issuerID)
```

Comment:
Same here 

Let's say it will be a key `authorization.text-field.issuer-id`
```suggestion
                AuthInputField(title: "Issuer ID", binding: $viewModel.issuerID)
```

Reply:
improved